src/platform/certs/zephyr/certs.c

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file      certs.c
+ * @brief     Mender MCU Certificate for zephyr
+ *
+ * Copyright Northern.tech AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "certs.h"
+#include "log.h"
+#include <errno.h>
+#include <zephyr/net/tls_credentials.h>
```

Comment:
We should `#include` third-party headers before our own headers. But not a biggie.